plugins/src/main/java/opengrok/auth/plugin/ldap/LdapServer.java

```diff
@@ -122,13 +129,89 @@ public void setInterval(int interval) {
         this.interval = interval;
     }
 
+    private String urlToHostname(String urlStr) throws URISyntaxException {
+        URI uri = new URI(urlStr);
```

Comment:
gotcha: cannot use `java.URL` since it does not recognize schemes other than http/https.